ansible_collections/arista/cvp/tests/inventory.ini

```diff
@@ -0,0 +1,36 @@
+[cvp:children]
+development
+lab
+
```

Comment:
We should remove references to internal project names (Foster / Grant)

Reply:
Solved with PR #88